src/sensor_state_data/sensor/device_class.py

```diff
@@ -15,6 +15,9 @@ class SensorDeviceClass(BaseDeviceClass):
     # % of battery that is left
     BATTERY = "battery"
 
+    # button (no unit)
+    BUTTON = "button"
```

Comment:
No, they aren’t. In BLE monitor, they are sensors that represent an event. E.g when a rocker switch is pressed long, it gets a state long press. A user can then create an automation, to do some action. 

Like these linptech wall switches for example
https://custom-components.github.io/ble_monitor/by_brand#linptech

these use the xiaomi MiBeacon protocol

Reply:
These should probably be represented as device triggers instead of sensors.

https://developers.home-assistant.io/docs/device_automation_trigger/
Examples:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/lutron_caseta/device_trigger.py
https://github.com/home-assistant/core/blob/dev/homeassistant/components/homekit_controller/device_trigger.py

Unfortunately we don't have a template for BLE devices yet.

Reply:
Hmm, yes, that would make sense. I’ll close this pr and look into that